pom.xml

```diff
@@ -0,0 +1,285 @@
+			<organizationUrl>https://ibissource.org</organizationUrl>
+		</developer>
+		<developer>
+			<name>Gerrit van Brakel</name>
+			<email>gerrit@ibissource.org</email>
+			<organization>Ibissource</organization>
+			<organizationUrl>https://ibissource.org</organizationUrl>
+		</developer>
+		<developer>
+			<name>Jaco de Groot</name>
+			<email>jaco@ibissource.org</email>
+			<organization>Ibissource</organization>
+			<organizationUrl>https://ibissource.org</organizationUrl>
+		</developer>
+	</developers>
+	<build>
+		<defaultGoal>install</defaultGoal>
+		<plugins>
+			<plugin>
+				<artifactId>maven-compiler-plugin</artifactId>
+				<version>3.8.1</version>
+				<configuration>
+					<source>1.8</source>
+					<target>1.8</target>
+				</configuration>
+			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-jar-plugin</artifactId>
+				<version>3.2.0</version>
+				<configuration>
+					<archive>
+						<addMavenDescriptor>true</addMavenDescriptor>
+						<manifest>
+							<addDefaultSpecificationEntries>true</addDefaultSpecificationEntries>
+							<addDefaultImplementationEntries>true</addDefaultImplementationEntries>
+						</manifest>
+					</archive>
+				</configuration>
+			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-source-plugin</artifactId>
+				<version>3.2.1</version>
+				<executions>
+					<execution>
+						<id>attach-sources</id>
+						<phase>prepare-package</phase>
+						<goals>
+							<goal>jar-no-fork</goal>
+						</goals>
+					</execution>
+				</executions>
+			</plugin>
+			<plugin>
+				<groupId>org.glassfish.copyright</groupId>
+				<artifactId>glassfish-copyright-maven-plugin</artifactId>
+				<version>2.4</version>
+				<executions>
+					<execution>
+						<?m2e ignore?> <!-- Eclipse doesn't seem to like this plugin, it doesn't need to be executed by Eclipse so ignore it -->
+						<id>Check-CopyRight-Notice</id>
+						<goals>
+							<goal>copyright</goal><!-- Checks for Copyright notices -->
+							<goal>check</goal><!-- Fails the build -->
+						</goals>
+						<phase>process-sources</phase>
+					</execution>
+				</executions>
+				<configuration>
+					<excludeFile>${project.basedir}/copyrightExcludeFile.txt</excludeFile>
+					<exclude>
+						DataSourceXAResourceRecoveryHelper.java,NarayanaRecoveryManager.java,FilenameUtils.java,PasswordHash.java
+					</exclude>
+					<ignoreYear>true</ignoreYear>
+					<useDash>true</useDash>
+					<templateFile>${project.basedir}/copyrightTemplateFile.txt</templateFile>
+				</configuration>
+			</plugin>
+			<plugin>
+				<artifactId>maven-surefire-plugin</artifactId>
+				<version>2.22.2</version>
+				<configuration>
+					<redirectTestOutputToFile>true</redirectTestOutputToFile>
+					<reuseForks>true</reuseForks>
+					<includes>
+						<include>**/*.java</include>
+					</includes>
+					<excludes>
+						<exclude>it.**</exclude>
+					</excludes>
```

Comment:
Deze exclude bestaat omdat wij in het framework een it package hebben die ge-exclude is. Deze is volgens mij niet nodig voor de frank-doc dus dan kan hij het beste weggehaald worden om zo ook verwarring te voorkomen.